exec/src/klio_exec/commands/run.py

```diff
@@ -552,6 +554,41 @@ def _setup_pipeline(self, pipeline):
                     **output_config.to_io_kwargs()
                 )
 
+    def wait_for_pipeline_running(
+        self,
+        result: PipelineResult,
+        timeout_sec=10 * 60,  # 10 minutes in seconds
+        poll_interval_sec=60,
+    ):
+        status = None
+
+        for _ in range(0, timeout_sec, poll_interval_sec):
+            try:
+                status = result.state
+                if status in PipelineState.RUNNING:
```

Comment:
That’s supposed to be status = PipelineState.RUNNING but I guess it would still work but I’ll fix this.